.github/workflows/qt_translation_update.yml

```diff
@@ -0,0 +1,77 @@
+name: Weekly Qt translation refresh
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'   # Runs every Sunday at 00:00 UTC
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+env:
+  CURRENT_BRANCH: ${{ github.ref_name }}
```

Comment:
and which branch is that? the default one (develop)?

Reply:
It's branch it which it runs. In this case Develop.